### PR TITLE
buyer_dashboard: include message if user has projects awaiting outcome information

### PR DESCRIPTION
**Pull request by risicle**
Trello https://trello.com/c/4Ub332Ty/70-fe-7-you-need-to-tell-us-the-outcome-for-1-saved-search-to-be-added-dynamic

I seem to have unconsciously added a full stop to the end of the sentence - @karlchillmaid is this desired or not?

after

### CODE REVIEW

File: tests/main/views/test_buyers.py [hunk 1]
```diff
@@ -2127,24 +2136,151 @@ def test_do_not_show_question_and_answer_dates_for_closed_brief(self):
         assert res.status_code == 404
 
 
+@mock.patch("app.main.views.buyers.data_api_client", autospec=True)
```
Comment (katstevens) [inline tests/main/views/test_buyers.py]: Noooooooo 🙀 Can we keep the api client patching in the setup method? This was part of the plan to move it all to a  setup method on the `BaseApplicationTest` helper 😿 😿 😿

Comment (risicle) [reply tests/main/views/test_buyers.py]: OH.

I wasn't aware of this and this isn't the only place I've made this transformation.

**Comment by karlchillmaid** [comment]
@risicle this is something that *keeps* coming up. It's basically a judgement call between pedants like me who think we _should_ use punctuation even if it looks ugly at the end of short sentences and sentences-that-end-in-links, and anyone who thinks we shouldn't.

Personally I think the full stop should stay. It was there in the original draft, but was removed to be consistent with all the other missing full stops.

If we do add it, we'd need to add one to the equivalent 'You have no saved searches' message – and do the same for the DOS messages.

There are probably many instances like this on DM, so we might be creating some content debt by adding a full stop in this case.

What do you think @tracy-content and @constancecerf?

**Comment by tracy-content** [comment]
@karlchillmaid back in the day making this consistent was in our content debt... This obviously didn't happen so I assume full stops have been inconsistently applied or not. 
What do we say in the style guide? If we don't say anything, maybe we should make a rule now and add it to the guide. Hopefully then we'll have a consistent approach from now on. 
I'd go for adding a full stop too!

**Comment by risicle** [comment]
O.k. I'm. putting. them. in. everywhere.

File: tests/main/views/test_notifications.py [hunk 2]
```diff
@@ -3,17 +3,9 @@
 from lxml import html
 
 
+@mock.patch('app.main.views.buyers.data_api_client', autospec=True)
```
[the remark this answers is not in the document]
Comment (risicle) [reply tests/main/views/test_notifications.py]: Yes it's just I wasn't aware of this but I thought I had now pushed the version that has it replaced as before...?

[the remark this answers is not in the document]
Comment (risicle) [reply tests/main/views/test_notifications.py]: Nope - hadn't, but have now.

[the remark this answers is not in the document]
Comment (katstevens) [reply tests/main/views/test_notifications.py]: Thanks for reverting - I meant to follow this up with a proper explanation but got caught up with other stuff. Happy to run through it if you like.

**Pull request merged**